octobot_trading/exchange_data/exchange_symbol_data.py

```diff
@@ -83,14 +90,30 @@ def handle_liquidations(self, liquidations):
         self.recent_trades_manager.add_new_liquidations(liquidations)
 
     def handle_mark_price_update(self, mark_price, mark_price_source) -> bool:
+        trigger_init_event = not self.prices_manager.initialized()
         updated = self.prices_manager.set_mark_price(mark_price, mark_price_source)
         if updated:
+            if trigger_init_event:
+                self._set_initialized_event(commons_enums.InitializationEventExchangeTopics.PRICE.value)
```

Comment:
👍 We can remove `price_events_manager.py ` now

Reply:
Can we ? to me these are different things

Reply:
If I remember well it's an equivalent of the new price event.